protocol/v2/qbft/controller/controller.go

```diff
@@ -37,14 +37,14 @@ type Controller struct {
 }
 
 func NewController(
-	identifier []byte,
+	Identifier func() []byte,
```

Comment:
A controller identifier should not change after its created, so I don't get why this func is needed.